util-test/src/commonMain/kotlin/care/data4life/sdk/util/test/CommonResourceLoader.kt

```diff
@@ -44,6 +44,8 @@ expect class CommonResourceLoader(
 
 // This should be scoped in the ResourceLoader and be private, however this is currently not supported by Kotlin
 internal object CommonPathResolver {
+    const val commonRoot: Path = "src/commonTest/resources"
```

Comment:
can this be private?

Reply:
Please look at the comment above.

Reply:
Or did you mean the prop? This can indeed be private.

Reply:
yeah i meant just the const

Reply:
done